projects/storefrontapp-e2e-cypress/cypress/helpers/address-book.ts

```diff
@@ -101,24 +109,28 @@ export function deleteExistingAddress() {
   let firstCard = cy.get('cx-address-card').first();
 
   firstCard.find('.delete').click();
-  cy.get('.cx-address-card-delete-msg').should(
-    'have.text',
-    ' Are you sure you want to delete this address? '
+  cy.get('.cx-address-card__delete-msg').contains(
```

Comment:
better use `.should(..)` instead of `.contains(...)` for an explicit assertion.

Reply:
Done!